src/client/styles/components/_tab-bar.scss

```diff
@@ -1,20 +1,22 @@
 .tab-bar {
+  $elem: &;
```

Comment:
This is totally clever but it took me a while to understand and I think the benefit is minimal— I'd prefer repetition of block names for the time being. (no idea if this is a `stylelint` change or a manual one)